tpm2/benchmark_test.go

```diff
@@ -0,0 +1,78 @@
+package tpm2
+
+import (
+	"crypto/sha256"
+	"math/big"
+	"testing"
+)
+
+func BenchmarkRSA2048Signing(b *testing.B) {
+	b.StopTimer()
+	rw := openTPM(b)
+	defer rw.Close()
+
+	pub := Public{
+		Type:       AlgRSA,
+		NameAlg:    AlgSHA256,
+		Attributes: FlagSign | FlagSensitiveDataOrigin | FlagUserWithAuth,
+		RSAParameters: &RSAParams{
+			Sign: &SigScheme{
+				Alg:  AlgRSASSA,
+				Hash: AlgSHA256,
+			},
+			KeyBits: uint16(2048),
+			Modulus: big.NewInt(0),
```

Comment:
I like what you did for ECPoint coords. Perhaps add the same helper to RSAParams.Modulus?

Reply:
The reason I did it here was that it was nested one layer down (through Point, which can also be nil). Modulus is accessed directly by RSAParams' encode() function. We could default the modulus to 0 there if we wanted, but right now there's explicit code forbidding that input (also because you can pass in the raw modulus).

Reply:
Ah, good point. nil Modulus actually has meaning when raw modulus is used